db/seeds.rb

```diff
@@ -5,3 +5,36 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+require 'faker'
+
+puts "will create trainer..."
+trainer = User.create!({
+  email: "trainer@mewbnb.com",
+  password: "password123"
+})
+
+puts "will create gym_leader..."
+gym_leader = User.create!({
+  email: "gymleader@mewbnb.com",
+  password: "password123"
+})
+
+
+10.times do
+  puts "will create pokemon..."
+  pokemon = Pokemon.create!({
```

Comment:
I think that you are missing 'level' and 'type' no?

Reply:
added level but for type i'm getting this error `ActiveRecord::SubclassNotFound: The single-table inheritance mechanism failed to locate the subclass: 'psychic'. This error is raised because the column 'type' is reserved for storing the class in case of inheritance. Please rename this column if you didn't intend it to be used for storing the inheritance class or overwrite Pokemon.inheritance_column to use another column for that information.
`

There's already a type column storing the data type internally so I'm gonna need to change the "type" column name to "pokemon_type" with a new migration. @jonathanjenin

Reply:
@randzoh I think it is because 'type' as a column name, is a reserved field. We would need to call the column something else. You may want to open a ticket to confirm it tho.